File: DAILY_CHALLENGE/MARCH_2024/25_03_2024.cpp
class Solution {
public:
    vector<int> findDuplicates(vector<int>& nums) {
        //int n=nums.size();
        sort(nums.begin(),nums.end());
        vector<int> v;
        int i=0;
        while(i<nums.size()-1)
        {
            if(nums[i]==nums[i+1])
            {
                v.push_back(nums[i]);
                i+=2;      
            }
            else{
                i++;
            }
        }
        return v;

        
    }
};

File: DAILY_CHALLENGE/MARCH_2024/25_03_2024.py
class Solution:
    def findDuplicates(self, nums: List[int]) -> List[int]:
        nums.sort();v=[];i=0

        while(i<len(nums)-1):
    
            if(nums[i]==nums[i+1]):
                v.append(nums[i])
                i+=2      
            
            else:
                i+=1  
        return v
